src/HealthCheckServiceProvider.php

```diff
@@ -4,24 +4,30 @@
 
 use Illuminate\Support\ServiceProvider;
 use UKFast\HealthCheck\Commands\CacheSchedulerRunning;
-use UKFast\HealthCheck\Controllers\PingController;
 use UKFast\HealthCheck\Controllers\HealthCheckController;
+use UKFast\HealthCheck\Controllers\PingController;
 
 class HealthCheckServiceProvider extends ServiceProvider
 {
     public function boot()
     {
         $this->configure();
+
+        $v = $this->app->make('router');
```

Comment:
I'm guessing you forgot to remove this during your testing.

```suggestion
```

Reply:
thx for the review. I changed the requested lines :)